docroot/modules/custom/cgov_glossary/cgov_glossary.install

```diff
@@ -0,0 +1,17 @@
+<?php
+
+use Drupal\app_module\Entity\AppModule;
+
+const MODULE_ID = 'cgov_glossary_app';
```

Comment:
BAH! This was a bad idea!  This is a _global_, and therefore in effect _everywhere_.....